definitions/xml2js.d.ts

```diff
@@ -0,0 +1,99 @@
+// Type definitions for node-xml2js
+// Project: https://github.com/Leonidas-from-XIV/node-xml2js
+// Definitions by: Michel Salib <https://github.com/michelsalib>, Jason McNeil <https://github.com/jasonrm>, Christopher Currens <https://github.com/ccurrens>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```

Comment:
Resolved, done - LCA approvals for xml2js, its three dependencies (xmlbuilder, lodash and sax), and the definitions file obtained.